2-ui/4-forms-controls/1-form-elements/article.md

```diff
@@ -182,7 +182,7 @@ A `<select>` element has 3 important properties:
 
 They provide three different ways of setting a value for a `<select>`:
 
-1. Find the corresponding `<option>` element (e.g. among `select.options`) and set its `option.selected` to `true`.
+1. Find the corresponding `<option>` element (e.g., among `select.options`) and set its `option.selected` to `true`.
```

Comment:
A comma here? Why?

Reply:
It is habit from a practice I've maintained.  The TL;DR of it is, a comma should follow an “e.g.”—but _not_ an “i.e.”.

Do you feel strongly about it?

Reply:
I almost never see e.g. with a comma. I'm not sure it's correct. Can you please remove the comma?

Reply:
Done!